app/src/main/res/values/dimens.xml

```diff
@@ -14,4 +14,6 @@
     <dimen name="dp_350">350dp</dimen>
     <dimen name="dp_30">30dp</dimen>
     <dimen name="dp_70">70dp</dimen>
+    <dimen name="dp_25">25dp</dimen>
+    <dimen name="sp_18">18sp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.sp_18` appears to be unused
 Unused resources make applications larger and slow down builds.